reward/reward_distributor.go

```diff
@@ -51,28 +53,41 @@ func NewRewardDistributor(gh governanceHelper) *RewardDistributor {
 }
 
 // getTotalTxFee returns the total transaction gas fee of the block.
-func (rd *RewardDistributor) getTotalTxFee(header *types.Header, rewardConfig *rewardConfig) *big.Int {
+func (rd *RewardDistributor) getTotalTxFee(header *types.Header, rewardConfig *rewardConfig, isKIP71Forked bool) (*big.Int, error) {
```

Comment:
1. The name of this method is `getTotalTxFe`, but your change burns tx fee as well as calculates tx fee. I think we better place the txx fee burn code outside of this method.
2. `rewardConfig` was used only to get `rewardConfig.unitPrice` which is a multiplier for tx fee calculation. I think we can modify the parameter to `gasPrice`, then this method can handle all cases before/after kip71 hardfork.

Reply:
Refactored it. Thanks.